<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConnect</class>
 <widget class="QDialog" name="DialogConnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="leditIP">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>IP地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leditPort">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>160</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>10001</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>54</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
